src/main/java/wooteco/subway/controller/StationController.java

```diff
@@ -0,0 +1,47 @@
+package wooteco.subway.controller;
+
+import java.net.URI;
+import java.util.List;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import wooteco.subway.controller.dto.request.station.StationCreateRequestDto;
+import wooteco.subway.controller.dto.response.station.StationResponseDto;
+import wooteco.subway.service.StationService;
+
+@RestController
+public class StationController {
+    private final StationService stationService;
+
+    public StationController(StationService stationService) {
+        this.stationService = stationService;
+    }
+
+    @PostMapping(value = "/stations",
+        consumes = MediaType.APPLICATION_JSON_VALUE,
+        produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<StationResponseDto> createStation(@RequestBody StationCreateRequestDto stationCreateRequestDto) {
+        StationResponseDto stationResponseDto = stationService.createStation(stationCreateRequestDto);
+        return ResponseEntity
+            .created(URI.create("/stations/" + stationResponseDto.getId()))
+            .body(stationResponseDto);
+    }
+
+    @GetMapping(value = "/stations", produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<List<StationResponseDto>> showAllStations() {
+        List<StationResponseDto> allStationResponses = stationService.getAllStations();
+        return ResponseEntity.ok()
+            .body(allStationResponses);
+    }
+
+    @DeleteMapping("/stations/{id}")
+    public ResponseEntity deleteStationById(@PathVariable Long id) {
```

Comment:
```suggestion
    public ResponseEntity<Void> deleteStationById(@PathVariable Long id) {
```

제네릭의 타입을 명시해주지 않을 경우 기능은 큰 차이가 없지만 컴파일 경고가 발생하게 되는데요!
컴파일 경고를 통해 매우 유용한 정보들을 많이 받을 수 있지만, 위와 같은 코드들로  불필요한 경고가 많아질 경우 개발자는 경고를 안보게 되는 문제가 발생합니다!
컴파일 시 경고가 발생한다면 바로바로 수정하는 습관을 들인다면 불필요한 버그를 줄이는데 도움이 될 것 같습니다 :)

Reply:
이 부분을 페어와도 얘기하다가, Body가 없으니 제네릭을 선언하지 않는 것이 맞지 않냐는 의견에 위와 같이 작성했어요. 재연링의 피드백 대로 Void가 있는 줄은 몰랐네요ㅠ. 좋은 피드백 감사합니다! :)